src/app/actions.ts

```diff
@@ -0,0 +1,13 @@
+'use server'
+
+import { createItem } from '@/lib/models/item'
+import { getCurrentAuthUser } from '@/lib/models/user'
+import { revalidatePath } from 'next/cache'
+
+export const addItemAction = async (formData: FormData): Promise<void> => {
```

Comment:
We can put all of our server actions here. If we create many of them, we may want to break them down into finer categories (similar to how the `/lib/models` directory is organized)